documentation/swagger-checklist.md

```diff
@@ -71,7 +71,7 @@ Where "M" stands for '*MUST*' or '*MUST NOT*' requirement. "S" stands for '*SHOU
 
 Today, there are tools available for validating a sizable portion of this checklist. Please install these tools on your development machine and use them to validate your Swaggers as you build them. 
 
--	[AutoRest Linter](https://github.com/Azure/autorest/wiki/linting) – Linter statically analyzes a Swagger for errors and violations of requirements outlined in this checklist. Linter should be run when Swaggers are created or updated. Linter will be continously updated to validate newer requirements.
+-	[AutoRest Linter](https://github.com/Azure/adx-documentation-pr/blob/master/tools/tools.md#semantic-validation) – Linter statically analyzes a Swagger for errors and violations of requirements outlined in this checklist. Linter should be run when Swaggers are created or updated. Linter will be continously updated to validate newer requirements.
```

Comment:
Would you please change the name of the section in tools.md to Linter (not semantic-validation)